CHANGES.md

```diff
@@ -25,6 +25,10 @@ OpenSSL 3.2
 
 ### Changes between 3.1 and 3.2 [xx XXX xxxx]
 
+ * Allow to disable HTTP support via the `no-http` configure option.
```

Comment:
"Allow to" sounds very clunky. How about

```
 * Provide a new configure option `no-http` configure option that can be used
   to disable HTTP support.
```

Reply:
Fixed, thanks !